test/service-test-utils/src/main/resources/META-INF/services/reactor.blockhound.integration.BlockHoundIntegration

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#  http://aws.amazon.com/apache2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+
+software.amazon.awssdk.testutils.service.BlockHoundAllowlist
```

Comment:
nit, should we move this to test directory?

Reply:
I believe it needs to be in this directory to be exported as part of the jar (which we want, so other modules pick up this configuration, namely the individual service IT modules).

https://stackoverflow.com/questions/17531625/how-to-include-a-config-file-in-the-meta-inf-services-folder-of-a-jar-using-ma

Reply:
Oh this is service-test-utils. NVM then!